src/components/CreateForm/styled.js

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+
+const FormContainer = styled.form`
```

Comment:
Maybe its better to name the FormContainer > StyledFormContainer, so you know that this is a styled component :)